test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpRazorCommentsTest/EmptyRazorComment.stree.txt

```diff
@@ -1,16 +1,12 @@
-Markup block - Gen<None> - 4 - (0:0,0)
-    Markup span - Gen<Markup> - [] - SpanEditHandler;Accepts:Any - (0:0,0) - Tokens:1
-        SyntaxKind.Unknown;[];
-    Comment block - Gen<RazorComment> - 4 - (0:0,0)
-        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (0:0,0) - Tokens:1
-            SyntaxKind.RazorCommentTransition;[@];
-        MetaCode span - Gen<None> - [*] - SpanEditHandler;Accepts:None - (1:0,1) - Tokens:1
-            SyntaxKind.RazorCommentStar;[*];
-        Comment span - Gen<None> - [] - SpanEditHandler;Accepts:Any - (2:0,2) - Tokens:1
-            SyntaxKind.Unknown;[];
-        MetaCode span - Gen<None> - [*] - SpanEditHandler;Accepts:None - (2:0,2) - Tokens:1
-            SyntaxKind.RazorCommentStar;[*];
-        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (3:0,3) - Tokens:1
-            SyntaxKind.RazorCommentTransition;[@];
-    Markup span - Gen<Markup> - [] - SpanEditHandler;Accepts:Any - (4:0,4) - Tokens:1
-        SyntaxKind.Unknown;[];
+RazorDocument - [0..4)::4 - [@**@]
+    MarkupBlock - [0..4)::4
+        MarkupTextLiteral - [0..0)::0 - [] - Gen<Markup> - SpanEditHandler;Accepts:Any
+            Marker;[];
+        RazorComment - [0..4)::4
+            RazorCommentTransition;[@];
+            RazorCommentStar;[*];
+            RazorCommentLiteral;[<Missing>];
+            RazorCommentStar;[*];
+            RazorCommentTransition;[@];
```

Comment:
I've simplified how Razor comments are represented. These are just tokens instead of being Spans.